Dockerfile

```diff
@@ -18,10 +18,12 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
     cd mlflow/server/js && \
     yarn install && \
     yarn build && \
-    # clean cache
+    # clean cache and logs
     apt-get autoremove -yqq --purge && apt-get clean && rm -rf /var/lib/apt/lists/* && \
     npm cache clean --force && \
     yarn cache clean --all && \
+    rm -rf /var/log/* && \
```

Comment:
How much does this line reduce the image size?

Reply:
About 3 megabytes.

Reply:
Do we really need this line?

Reply:
I agree that it is not enough, but it still saves free space.